.changeset/spicy-masks-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/analytics": patch 
+---
+
+Browser Extension Check for Analytics Module
```

Comment:
Can you make it a bit more specific? Something like: `analytics.isSupport() will now return false for extension environments`

Reply:
Can you add `firebase: patch` too, so we will generate changelog for `firebase`?